Content.Server/Supermatter/Components/SupermatterComponent.cs

```diff
@@ -0,0 +1,342 @@
+        public static float MIASMA_GASMIX_SCALING = (0.3f);
+        // The amount of matter power generated for every mole of miasma consumed. Higher values mean miasma generates more power.
+        public static float MIASMA_POWER_GAIN = 10f;
+
+        //Higher == Higher percentage of inhibitor gas needed before the charge inertia chain reaction effect starts.
+        public static float  PowerlossInhibitionGasThreshold = 0.20f;
+        //Higher == More moles of the gas are needed before the charge inertia chain reaction effect starts.        //Scales powerloss inhibition down until this amount of moles is reached
+        public static float  PowerlossInhibitionMoleThreshold = 20f;
+        //bonus powerloss inhibition boost if this amount of moles is reached
+        public static float  PowerlossInhibitionMoleBoostThreshold = 500f;
+        //Above this value we can get lord singulo and independent mol damage, below it we can heal damage
+        public static float  MolePenaltyThreshold = 1800f;
+
+        //Along with damage_penalty_point, makes flux anomalies.
+        /// The cutoff for the minimum amount of power required to trigger the crystal invasion delamination event.
+        public static float  EVENT_POWER_PENALTY_THRESHOLD = 4500f;
+        //The cutoff on power properly doing damage, pulling shit around, and delamming into a tesla. Low chance of pyro anomalies, +2 bolts of electricity
+        public static float  PowerPenaltyThreshold = 5000f;
+        //+1 bolt of electricity, allows for gravitational anomalies, and higher chances of pyro anomalies
+        public static float  SEVERE_POWER_PENALTY_THRESHOLD = 7000f;
+        //+1 bolt of electricity.
+        public static float  CRITICAL_POWER_PENALTY_THRESHOLD = 9000f;
+        //Higher == Crystal safe operational temperature is higher.
+        public static float  HeatPenaltyThreshold = 40f;
+        public static float  DamageHardcap = 0.002f;
+        public static float  DamageIncreaseMultiplier = 0.25f;
+
+        //Higher == less heat released during reaction, not to be confused with the above values
+        public static float  ThermalReleaseModifier = 5f;
+        //Higher == less plasma released by reaction
+        public static float  PlasmaReleaseModifier = 750f;
+        //Higher == less oxygen released at high temperature/power
+        public static float  OxygenReleaseModifier = 325f;
+        //Higher == more overall power
+        public static float  ReactionPowerModefier = 0.55f;
+        //Crystal converts 1/this value of stored matter into energy.
+        public static float  MATTER_POWER_CONVERSION = 10f;
+        //These would be what you would get at point blank, decreases with distance
+        public static float  DETONATION_RADS = 200f;
+        public static float  DETONATION_HALLUCINATION = 600f;
+
+        public static float MaxSpaceExposureDamage = 2;
```

Comment:
Make all of these constants and move them to the `Atmospherics` static class in `Content.Shared` instead...

Reply:
I would say leave all this in here instead of flooding the Atmospherics class as they where all in the original tg Supermatter define and are only used in the Supermatter.

Reply:
You can use partials for it.